backends/advanced/src/advanced_omi_backend/routers/modules/obsidian_routes.py

```
@@ -0,0 +1,227 @@
+            try:
+                job_data = json.loads(pending_data)
+                return {
+                    "job_id": job_id,
+                    "status": "ready",
+                    "total": job_data.get("total", 0),
+                    "processed": 0,
+                    "percent": 0,
+                    "errors": [],
+                    "vault_path": job_data.get("vault_path")
+                }
+            except:
```

Ruff (0.14.8)

Do not use bare `except`

(E722)